src/supermarket/config/initializers/paperclip.rb

```diff
@@ -21,7 +21,8 @@
       },
       path: path,
       bucket: ENV['S3_BUCKET'],
-      s3_protocol: ENV['PROTOCOL']
+      s3_protocol: ENV['PROTOCOL'],
+      s3_region: ENV['S3_REGION']
```

Comment:
Confirmed with @nathenharvey that the S3 settings must be absent or complete else the service should fail to start and provide useful feedback